.eslintrc.js

```diff
@@ -7,9 +7,9 @@ module.exports = {
     'eslint:recommended',
     'plugin:@typescript-eslint/eslint-recommended',
     'plugin:@typescript-eslint/recommended-requiring-type-checking',
-    'plugin:@angular-eslint/recommended'
+    'plugin:@angular-eslint/recommended',
   ],
-  plugins: ['rxjs'],
+  plugins: ['rxjs', 'jest'],
```

Comment:
Is it normal to add a Jest ESLint plugin when we actually don't use Jest … ?

Reply:
I did some digging, I think we are using jasmine, there's an ESLint jasmine plugin with a rule that looks to do the trick:
https://github.com/tlvince/eslint-plugin-jasmine/blob/master/docs/rules/no-focused-tests.md